setup.cfg

```diff
@@ -0,0 +1,2 @@
+[bdist_wheel]
```

Comment:
`[wheel]` is legacy https://bitbucket.org/pypa/wheel/src/4da780b849affbff98a0defb21ad1f30a94d6f57/wheel/bdist_wheel.py?at=default&fileviewer=file-view-default#bdist_wheel.py-119